routemaster/conftest.py

```diff
@@ -373,13 +373,17 @@ def _delete(name: str, state_machine_name: str) -> None:
 
 
 @pytest.fixture()
-def create_deleted_label(create_label, delete_label):
+def create_deleted_label(create_label, delete_label, metadata={}):
     """
     Create a label in the database and then delete it.
     """
 
-    def _create_and_delete(name: str, state_machine_name: str) -> LabelRef:
-        create_label(name, state_machine_name, {})
+    def _create_and_delete(
+        name: str,
+        state_machine_name: str,
+        metadata: dict = {},
```

Comment:
Please could we type the metadata value more correctly? `dict` isn't really a valid type annotation (we should prefer a [generic `Dict`](https://docs.python.org/3/library/typing.html#typing.Dict) in general) and in this case I think we have a `Metadata` type somewhere?